lib/const.rb

```diff
@@ -96,7 +96,7 @@
 CREW_CONFIG_PATH      = File.join(CREW_PREFIX, 'etc/crew/')
 CREW_META_PATH        = File.join(CREW_CONFIG_PATH, 'meta/')
 CREW_BREW_DIR         = File.join(CREW_PREFIX, 'tmp/crew/')
-CREW_DEST_DIR         = File.join(CREW_BREW_DIR, 'dest/')
+CREW_DEST_DIR         = File.join(CREW_BREW_DIR, 'dest')
 CREW_WINE_PREFIX      = File.join(CREW_LIB_PREFIX, 'wine/')
```

Comment:
All of these constants are currently using trailing slashes.  We should be consistent throughout with whatever we decide.

Reply:
Agreed, but I'm only changing one at a time to make sure the unit tests pass...